start.py

```diff
@@ -293,6 +293,19 @@ def set_jvm_memory(m2ee_section, vcap, java_version):
         )
 
 
+def set_jetty_config(m2ee):
+    jetty_config_json = os.environ.get('JETTY_CONFIG')
+    if not jetty_config_json:
+        return None
+    try:
+        jetty_config = json.loads(jetty_config_json)
+        jetty = m2ee.config._conf['m2ee']['jetty']
+        jetty.update(jetty_config)
+        logger.debug('Jetty configured: %s ' % json.dumps(jetty))
+    except Exception as e:
+        logger.warning('Failed to configure jetty: ' + str(e))
```

Comment:
You can do this better in the following way:
```
logger.warning("Failed to configure jetty.", exc_info=True)
```
Because that includes not only the string representation of the exception, but also the traceback.